dev/com.ibm.ws.cdi.web/src/com/ibm/ws/cdi/web/impl/AbstractInitialListenerRegistration.java

```diff
@@ -89,7 +89,7 @@ public void registerListener(IServletContext isc) {
                     Tr.debug(tc, "added WeldInitialListener to the servlet context");
 
                 //Put bean manager on the servlet context
-                isc.setAttribute("javax.enterprise.inject.spi.BeanManager", beanManager);
+                isc.setAttribute(BeanManager.class.getCanonicalName(), beanManager);
```

Comment:
getName() is a much simpler method to call and there is no reason to require getCanonicalName

Reply:
Changed.